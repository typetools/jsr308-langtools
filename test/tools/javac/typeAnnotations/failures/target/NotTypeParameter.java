/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary test invalid location of TypeUse
 * @author Mahmood Ali
 * @compile/fail/ref=NotTypeParameter.out -XDrawDiagnostics -source 1.8 NotTypeParameter.java
 */

import java.lang.annotation.Target;
import java.lang.annotation.ElementType;

class VoidMethod<@A K> {
  @A void test() { }
}

@Target(ElementType.TYPE_USE)
@interface A { }
