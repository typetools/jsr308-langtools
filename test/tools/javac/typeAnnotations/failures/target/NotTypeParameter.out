NotTypeParameter.java:35:3: compiler.err.annotation.type.not.applicable
NotTypeParameter.java:34:18: compiler.err.annotation.type.not.applicable
2 errors
